Frontend/ui-driver/src/Screens/HomeScreen/ComponentConfig.purs

```diff
@@ -1020,6 +1020,35 @@ rateCardState state =
   in
     rateCardConfig'
 
+interCityRateCard :: ST.HomeScreenState -> RateCard.Config
+interCityRateCard state =
+ let
+    config' = RateCard.config
+    interCityRateCard' =
+      config'
+        { title = "RATE CARD"
+        , buttonText = Nothing
+        , currentRateCardType = CommonTypes.PaymentFareBreakup
+        , primaryButtonConfig {
+            text = getString GOT_IT
+          , visibility = VISIBLE
+          }
+        , fareList = [
+          {key : "DISTANCE_FARE",val : "₹" <> "0.0"},
```

Comment:
why are we sending these values as 0.0?

Reply:
Because currently  we are not getting them from backend

Reply:
Remove the rate card until you get proper data from backend.
cc: @Karthickselvan28